action/rlp_tx.go

```diff
@@ -83,3 +82,68 @@ func DecodeRawTx(rawData string, chainID uint32) (tx *types.Transaction, sig []b
 	pubkey, err = crypto.RecoverPubkey(rawHash[:], sig)
 	return
 }
+
+// DecodeEtherTx decodes raw data string into eth tx
+func DecodeEtherTx(rawData string) (*types.Transaction, iotextypes.Encoding, []byte, crypto.PublicKey, error) {
+	var (
+		tx       types.Transaction
+		encoding iotextypes.Encoding
+		sig      []byte
+		pubkey   crypto.PublicKey
+	)
+	//remove Hex prefix and decode string to byte
+	rawData = strings.Replace(rawData, "0x", "", -1)
+	rawData = strings.Replace(rawData, "0X", "", -1)
+	rawTxBytes, err := hex.DecodeString(rawData)
+	if err != nil {
+		return &tx, encoding, sig, pubkey, err
+	}
+
+	// decode raw data into rlp tx
+	if err = tx.UnmarshalBinary(rawTxBytes); err != nil {
+		return &tx, encoding, sig, pubkey, err
+	}
+
+	// extract correct V value
+	var (
+		signer  types.Signer
+		V, R, S = tx.RawSignatureValues()
```

Comment:
v, r, s

Reply:
this is internal var so should be OK? and below we have
```
r, s := R.Bytes(), S.Bytes()
copy(sig[32-len(r):32], r)
copy(sig[64-len(s):64], s)
```